ios/dashboard.html

```diff
@@ -97,6 +98,20 @@
 <p>We've made &pound;<span id="revenueThisMonth"></span> this month</p><script src="revenueThisMonth.js"></script>
 <div id="revenue6Months" style="min-width: 310px; max-width: 940px; height: 400px; margin: 0; border-bottom:2px #d3d3d3 solid;"></div><script src="revenue6Months.js"></script>
     </div><!--/.revenue-->
+    <div class="page" id="metrics">
+            <div class="row" style="margin-top: 30px">
+                <div class="col-md-4">
+                 <div id="starRatingsGraph" style="width:100%; height:400px;"></div>
```

Comment:
Might need to revisit how the width and height are set later on, when there's more than one chart.